uwblueprint/ccbc
# Kenson/creator profile presentations form

## Notion ticket link

[Creator Profile Presentation Form](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
Added Presentation type to CreatorTypes, creatorProfile has presentations attribute which is an array of these presentations.
Missing features:
- Photo upload
- Richtext
- Form validation
- In-person/virtual buttons and inputs
- New presentations button
- Past presentations setup with the backend must be redone with new fields



## Steps to test
1. Go to /finish-profile
2. Go to presentaiton form
3.  Test each input field to ensure they work



## What should reviewers focus on?
* 


## Checklist
